atlp-rwanda/atlp-pulse-fn
# bg [fix]-landing page

# PR Description #287 
This pull request addresses the style fixes for the landing page as described in Issue #287. The changes aim to improve the visual appeal and user experience of our landing page by standardizing typography, enhancing the color scheme, refining layout and spacing, optimizing images, and improving the design of call-to-action buttons.


# Description of tasks that were expected to be completed
- Standardized font sizes, line heights, and font styles for consistent typography.
- Aligned the color scheme with branding guidelines for a cohesive look.
- Improved overall layout and spacing for better visual appeal and readability.
- Optimized images for faster page load times and responsiveness.
- Standardized the design of call-to-action buttons for a more engaging user experience.
- Addressed alignment issues for responsive design.
- Refined text formatting for improved readability.

# How has this been tested?
I tested the changes on various devices and screen sizes to ensure responsive design and proper alignment. I also verified that the color scheme and typography align with our branding guidelines. Additionally, I ran performance tests to confirm that the image optimization improved page load times.

To reproduce the tests:
1. Open the landing page on different devices (desktop, tablet, mobile).
2. Verify that the layout and alignment are consistent.
3. Check the color scheme against branding guidelines.
4. Measure the page load times before and after image optimization.

# Number of Commits
This pull request consists of 2 commits, which have been squashed for clarity.

# Screenshots (If appropriate)
![Screenshot 2023-09-06 135916](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/107407670/1bd50bbf-8dec-42c7-a2ef-9d9df73b8060)


# Please check this Checklist before you submit your PR:

- [x] My test coverage meets the set test coverage threshold
- [x] Testimonial responsive fix
- [x]  fix for about us card
